settings.py

```diff
@@ -491,6 +491,7 @@
 DAILY_DEPLOY_EMAIL = None
 EMAIL_SUBJECT_PREFIX = '[commcarehq] '
 SAAS_REPORTING_EMAIL = None
+INTERNAL_FEEDBACK_EMAIL = 'internal-feedback@example.com'
```

Comment:
we don't need this now yes?

Reply:
We don't, I'll remove this ([1d22c49](https://github.com/dimagi/commcare-hq/pull/34026/commits/1d22c49ab148ea3cb98b77a1adadc10bcf9fea75))